src/app/shared/layout/card/card.module.ts

```diff
@@ -1,17 +1,14 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { MatIconModule } from '@angular/material/icon';
+import { SharedModule } from '../../shared/shared.module';
 import { CardComponent } from './card.component';
 
 export const cardSchema: NgModule = {
   declarations: [CardComponent],
-  imports: [CommonModule, MatIconModule],
+  imports: [CommonModule, MatIconModule, SharedModule],
```

Comment:
Is there a way to just import the directive instead of the full module?

Reply:
I can make it into its own module, do you want it to stay in the shared folder?